### PR TITLE
mgr/telemetry: remove aggregated perf metrics from the perf channel

### PR DESCRIPTION
**Pull request by ljflores**
Up until this point, we included aggregated and separated data for
testing purposes. Now that we've done our testing, the aggregated
metrics are no longer relevant.

Aggregated metrics can still be achieved on the server side by
summing separated metrics.

Signed-off-by: Laura Flores <lflores@redhat.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [x] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests
- Teuthology
  - [ ] Completed teuthology run
  - [ ] No teuthology test necessary (e.g., documentation)

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by ljflores** [comment]
jenkins retest this please

**Comment by neha-ojha** [comment]
jenkins retest this please

**Review by nizamial09 — APPROVED** [review]

File: src/pybind/mgr/telemetry/module.py [hunk 1]
```diff
@@ -900,19 +900,11 @@ def compile_report(self, channels: Optional[List[str]] = None) -> Dict[str, Any]
             report['crashes'] = self.gather_crashinfo()
 
         if 'perf' in channels:
-            report['perf_counters_aggregated'] = self.gather_perf_counters('aggregated')
-            report['perf_counters_separated'] = self.gather_perf_counters('separated')
-
-            report['stats_per_pool'] = self.get('pg_dump')['pool_stats']
```
Comment (yaarith) [inline src/pybind/mgr/telemetry/module.py:906]: can stats_per_pool be derived from stats_per_pg below? if not, we need to decide what metrics to keep here

Comment (ljflores) [reply src/pybind/mgr/telemetry/module.py:906]: Thanks for pointing this out, I was too rash in removing `stats_per_pool` altogether.

I've been working on compiling a spreadsheet of all the perf channel metrics. If you check [this spreadsheet](https://docs.google.com/spreadsheets/d/1VBkQD_Jknxqx7R_8n0-XZqZ13QgP3PBQlYH1XBOiRh0/edit?usp=sharing), you can see a comparison of metrics between `stats_per_pg` and `stats_per_pool`.

In summary, we can derive these `stats_per_pool` metrics from `stats_per_pg`:
- `log_size`
- `num_pg`
- `ondisk_log_size`
- `poolid`
- `stat_sum`
We cannot derive these metrics from `stats_per_pg`, as they are specific to `stats_per_pool`:
- `acting`
- `num_store_stats`
- `store_stats`
- `up`

We could remove the redundant metrics from `stats_per_pool`, but it also might make sense to keep them there to make it easier on someone inspecting the report.

Comment (ljflores) [reply src/pybind/mgr/telemetry/module.py:906]: Added the `stats_per_pool` metrics back in for now.

**Comment by ljflores** [comment]
jenkins retest this please

**Comment by ljflores** [comment]
jenkins test make check arm64

**Comment by ljflores** [comment]
jenkins retest this please

**Review by yaarith — APPROVED** [review]

**Review by alfonsomthd — APPROVED** [review]

**Comment by ljflores** [comment]
http://pulpito.front.sepia.ceph.com/yuriw-2021-12-09_00:18:57-rados-wip-yuri-testing-2021-12-08-1336-distro-default-smithi

Failures, unrelated:

    6553716, 6553740, 6553788, 6553822, 6553844, 6553876, 6553930, 6553953, 6553982, 6554000, 6554035, 6554063, 6554085 -- https://tracker.ceph.com/issues/53487
    6553768, 6553897 -- failed in recent master baseline: http://pulpito.front.sepia.ceph.com/yuriw-2021-12-07_00:28:11-rados-wip-master_12.6.21-distro-default-smithi/6549263/
    6553774 -- https://tracker.ceph.com/issues/50280
    6553780, 6553993 -- https://tracker.ceph.com/issues/53499
    6553781, 6553994 -- https://tracker.ceph.com/issues/53501
    6554077 -- https://tracker.ceph.com/issues/51904
    6553724 -- https://tracker.ceph.com/issues/52657
    6553853 -- infrastructure failure

Details:

Bug_#53487: qa: mount error 22 = Invalid argument - Ceph - CephFS
Bug_#50280: cephadm: RuntimeError: uid/gid not found - Ceph
Bug_#53499: testdashboard_e2e.sh Failure: orchestrator/02-hosts-inventory.e2e failed. - Ceph - Mgr - Dashboard
Bug_#53501: Exception when running 'rook' task. - Ceph - Orchestrator
Bug_#51904: AssertionError: wait_for_clean: failed before timeout expired due to down PGs - Ceph - RADOS
Bug_#52657: MOSDPGLog::encode_payload(uint64_t): Assertion `HAVE_FEATURE(features, SERVER_NAUTILUS)' - Ceph - RADOS

**Pull request merged**